### ISSUE TITLE
Allow horizontal movement speed to go below 0.01

### ISSUE THREAD

**Issue by ruguysgoingtrickortreating** [PssbleTrngle/CreateJetpack#111]
### Describe the feature you have in mind.

I would like the jetpack to be purely a building tool on my server, but the fact that movement speed can't go below 0.01 gives more mobility than I'd like. Could we go below this cap to further hinder horizontal movement speed?

### Does this solve a particular problem?

Yes, it discourages using the jetpack to get places and encourages using trains.

**Comment by ruguysgoingtrickortreating** [comment]
Alternatively, disabling sprint while jetpacked would also be good

**Comment by Batres3** [comment]
Yes, also it makes no sense that setting it to 1 gives ridiculous flight speeds, it would make more sense for 1 to be regular walk speed, and then 0.01 to be 1% of that.

**Comment by Batres3** [comment]
Also just noting that this is as simple as editing the file `ServerConfig.kt` line 42.
